packages/contract/lib/contract/properties.js

```diff
@@ -232,7 +232,7 @@ module.exports = {
 
         signature += ")";
 
-        var topic = web3Utils.keccak256(signature);
+        var topic = web3Utils.sha3(signature);
```

Comment:
We [export keccak256](https://github.com/web3/web3.js/blob/47abef004ea075c74fcf6526da7d6bf7275b6c9e/packages/web3-utils/src/hash.ts#L141), so this change shouldn't be necessary. Additionally, sha3 and keccak256 are different and will return different hashes (@spacesailor24)

Reply:
fixed